Makefile

```diff
@@ -11,10 +11,10 @@ fmt:
 	cargo +nightly fmt --all
 
 clippy_check:
-	cargo clippy --workspace  --all-targets -- -D warnings
+	cargo clippy --workspace  --all-targets --features coordinator_e2e_test -- -D warnings
 
 clippy:
-	cargo clippy --workspace  --all-targets --fix --allow-staged
+	cargo clippy --workspace  --all-targets --features coordinator_e2e_test --fix --allow-staged
```

Comment:
coordinator_e2e_test or meta_e2e_test?

Reply:
At present, I have encountered the clippy problem of `coordinator_e2e_test`. It is estimated that `coordinator_e2e_test` and `meta_e2e_test` also has problems, added to the `make clippy_check` later